sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/Group.java

```diff
@@ -1033,6 +1037,9 @@ static <InputT> CombineFieldsByFields<InputT> of(
           .setSchemaAggregateFn(schemaAggregateFn)
           .setKeyField(keyField)
           .setValueField(valueField)
+          .setFewKeys(
+              true) // We are often selecting only certain fields, so combiner lifting usually
```

Comment:
will this break update compatibility for jobs already using Group transform?

Reply:
Yes, however there is a workaround (call withPrecombining(false)). IMO we should document this in the Beam release notes along with the workaround.

Reply:
sgtm